tree/ntupleutil/v7/test/ntuple_importer.cxx

```diff
@@ -1,8 +1,372 @@
+      jet_eta[0] = 2.0;
+      muon_pt[0] = 10.0;
+      tree->Fill();
+      njets = 0;
+      muon_pt[0] = 11.0;
+      tree->Fill();
+      njets = 2;
+      jet_pt[0] = 3.0;
+      jet_eta[0] = 4.0;
+      jet_pt[1] = 5.0;
+      jet_eta[1] = 6.0;
+      muon_pt[0] = 12.0;
+      tree->Fill();
+      tree->Write();
+   }
+
+   auto importer = RNTupleImporter::Create(fileGuard.GetPath(), "tree", fileGuard.GetPath()).Unwrap();
+   importer->SetIsQuiet(true);
+   importer->SetNTupleName("ntuple");
+   importer->Import();
+
+   auto reader = RNTupleReader::Open("ntuple", fileGuard.GetPath());
+   EXPECT_EQ(3U, reader->GetNEntries());
+   auto viewBegin = reader->GetView<std::int32_t>("begin");
+   auto viewMiddle = reader->GetView<std::int32_t>("middle");
+   auto viewEnd = reader->GetView<std::int32_t>("end");
+   EXPECT_EQ(1, viewBegin(0));
+   EXPECT_EQ(2, viewMiddle(0));
+   EXPECT_EQ(3, viewEnd(0));
+   auto viewJets = reader->GetViewCollection("_collection0");
+   auto viewJetPt = viewJets.GetView<float>("jet_pt");
+   auto viewJetEta = viewJets.GetView<float>("jet_eta");
+   auto viewMuons = reader->GetViewCollection("_collection1");
+   auto viewMuonPt = viewMuons.GetView<float>("muon_pt");
+   EXPECT_EQ(1, viewJets(0));
+   EXPECT_FLOAT_EQ(1.0, viewJetPt(0));
+   EXPECT_FLOAT_EQ(2.0, viewJetEta(0));
+   EXPECT_EQ(0, viewJets(1));
+   EXPECT_EQ(2, viewJets(2));
+   EXPECT_FLOAT_EQ(3.0, viewJetPt(1));
+   EXPECT_FLOAT_EQ(4.0, viewJetEta(1));
+   EXPECT_FLOAT_EQ(5.0, viewJetPt(2));
+   EXPECT_FLOAT_EQ(6.0, viewJetEta(2));
+   EXPECT_EQ(1, viewMuons(0));
+   EXPECT_EQ(1, viewMuons(1));
+   EXPECT_EQ(1, viewMuons(2));
+   EXPECT_FLOAT_EQ(10.0, viewMuonPt(0));
+   EXPECT_FLOAT_EQ(11.0, viewMuonPt(1));
+   EXPECT_FLOAT_EQ(12.0, viewMuonPt(2));
+}
+
+TEST(RNTupleImporter, STL)
+{
+   FileRaii fileGuard("test_ntuple_importer_stl.root");
+   {
+      std::unique_ptr<TFile> file(TFile::Open(fileGuard.GetPath().c_str(), "RECREATE"));
+      auto tree = std::make_unique<TTree>("tree", "");
+      auto vec = new std::vector<float>{1.0, 2.0};
+      tree->Branch("vec", &vec);
+      tree->Fill();
+      tree->Write();
+      delete vec;
+   }
+
+   auto importer = RNTupleImporter::Create(fileGuard.GetPath(), "tree", fileGuard.GetPath()).Unwrap();
+   importer->SetIsQuiet(true);
+   importer->SetNTupleName("ntuple");
+   importer->Import();
+   auto reader = RNTupleReader::Open("ntuple", fileGuard.GetPath());
+   EXPECT_EQ(1U, reader->GetNEntries());
+   reader->LoadEntry(0);
+
+   auto vec = reader->GetModel()->Get<std::vector<float>>("vec");
+   EXPECT_EQ(2U, vec->size());
+   EXPECT_FLOAT_EQ(1.0, vec->at(0));
+   EXPECT_FLOAT_EQ(2.0, vec->at(1));
+}
+
+TEST(RNTupleImporter, CustomClass)
+{
+   FileRaii fileGuard("test_ntuple_importer_custom_class.root");
+   {
+      std::unique_ptr<TFile> file(TFile::Open(fileGuard.GetPath().c_str(), "RECREATE"));
+      auto tree = std::make_unique<TTree>("tree", "");
+      CustomStructUtil *klass = nullptr;
```

Comment:
```suggestion
      CustomStructUtil *object = nullptr;
```
('klass' sounds like 'class' which sounds like it is meta data about the class itself).